kapitan/targets.py

```diff
@@ -349,6 +349,25 @@ def valid_target_obj(target_obj):
                     },
                     "required": ["input_type", "input_paths", "output_path"],
                     "minItems": 1,
+                    "oneOf": [
+                        {
+                            "properties": {
+                                "input_type": {
+                                    "enum": ["jsonnet", "kadet"] 
+                                },
+                                "output_type" : {
+                                    "enum": ["yaml", "json"]
```

Comment:
we also now support "plain" output_type for all input_types, so please make sure to include that as well please.

Reply:
Added.